apps/journeys-admin/src/components/ReportButtons/ReportButtons.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent,waitFor } from '@testing-library/react'
+import { MockedProvider } from '@apollo/client/testing'
+import { SnackbarProvider } from 'notistack'
+import { FlagsProvider } from '@core/shared/ui/FlagsProvider'
+import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
+import { ReportButtons } from './ReportButtons'
+
+
+describe('ReportButtons', () => {
+  it('should have Journeys button with push to journeys', async () => {
+    const push = jest.fn()
```

Comment:
```suggestion
    const push = jest.fn()
    mockUseRouter.mockReturnValue({ push } as unknown as NextRouter)
```

need to use mock return the value from the router here